### PR TITLE
Remove get_hardware_capablities() and 1ph3ph switching support

### PR DESCRIPTION
**Pull request by corneliusclaussen**
Implementation of evse_board_support was simplified a bit. The sync command to get hw capablities on startup is no longer needed as the async publish of capablities is enough. Each BSP implementation module is required to publish its capablities at least once in the ready function. It may publish it later on to update the values e.g. for thermal derating.

1ph/3ph switching support during charging with a hyteresis if the 1ph/3ph power ranges overlap (e.g. on 22kW).
Hysteresis only really works if EV and EVSE have the same max current rating, e.g. both 16A See doc.rst for details.

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have made corresponding changes to the documentation
- [X] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements

### CODE REVIEW

File: modules/YetiDriver/yetiR1_2.2_firmware.bin
Comment (Pietfried) [inline modules/YetiDriver/yetiR1_2.2_firmware.bin:1]: This requires a change here as well
https://github.com/EVerest/everest-core/blob/main/modules/YetiDriver/CMakeLists.txt#L39

Comment (corneliusclaussen) [reply modules/YetiDriver/yetiR1_2.2_firmware.bin:1]: fixed

File: modules/EvseManager/EvseManager.hpp [hunk 1]
```diff
@@ -231,6 +235,7 @@ class EvseManager : public Everest::ModuleBase {
             charger->inform_new_evse_max_hlc_limits(evseMaxLimits);
         }
     }
+    std::atomic_int ac_nr_ph_phases_active{0};
```
Comment (Pietfried) [inline modules/EvseManager/EvseManager.hpp:238]: `ac_nr_phases_active` ?

File: modules/PhyVersoBSP/connector_1/evse_board_supportImpl.cpp [hunk 2]
```diff
@@ -59,15 +59,6 @@ void evse_board_supportImpl::init() {
 void evse_board_supportImpl::ready() {
 }
 
-void evse_board_supportImpl::handle_setup(bool& three_phases, bool& has_ventilation, std::string& country_code) {
```
Comment (Pietfried) [inline modules/PhyVersoBSP/connector_1/evse_board_supportImpl.cpp:62]: Looks like you need to publish the caps for connector 1 as well like its done for connector 2

**Review by Pietfried — CHANGES_REQUESTED** [review]
See comments + Lint and Tests are currently failing

**Review by Pietfried — APPROVED** [review]

**Pull request merged**